### ISSUE TITLE
Create a future system for working groups to demo internal findings with each other. 

### ISSUE THREAD

**Issue by mrmetaverse** [omigroup/OMI#59]
## How might we share our recent findings with each other in an engaging way? 
Organize and Implement process for show + tell by contributors of OMI, their respective orgs and affiliated working groups to discuss potential use cases, lessons learned, and blockers to solve for. (this is separate from ~~lightning talks~~ official public event Demo Nights) Essentially we can expect to have ongoing demos to discuss technical details and discoveries, which we hope will promote ad-hoc meetings for cross-pollination (as needed) empowering collaboration. We hope this will boost our community. This is a part of working groups establishing their own meeting cadences. Some groups may want to demo their work every month, some may want to do more or less frequent. 

💡**This ticket is about identifying an initial process, not about planning any one specific event. If you are interested in speaking at a Demo day, feel free to leave a comment here, and we will provide you with the decided method sign-up.**  

## Acceptance Criteria: 
- Assemble an ad hoc team to explore and plan _the process._ 
- Identify a method for volunteering to share at demo day (do we want a sign up form? Do we create github discussion pages for signing up and reflecting on the meeting [like we do with our weekly meetings](https://github.com/omigroup/OMI/discussions/54)?  or do we want to be more informal and on the spot) 
- How might we document these lessons learned on our github / omigroup website. 
- Set a date for the first demo day
- Establish alternating time zones 

Keep in mind the process is flexible and can change. We should look to start simple.

**Comment by jsealt** [comment]
In yesterdays Fiscal development meeting, the group decided on two potential dates for the first demo day. 
- August 19th  or August 26th (Thursdays) 

- Lightweight event ~1 month from now
    - OMI Demo Day
    - Possibly taking place in Jel
        - Pros:
            - High room capacity
            - Screen sharing
        - Cons:
            - Voxelized glTFs for artists
            - Hopping between experiences lets users see them first hand
        - August 19th or August 26th
- Larger event in October
    - Art Day / Hackathon

- https://hackmd.io/@robertlong/omi-fiscal-development
- https://github.com/omigroup/OMI/discussions/67?converting=1#discussioncomment-1039451

**Comment by mrmetaverse** [comment]
Reminder, there is a fundamental difference between demo days and Lightning Talks. 

Demo days are a part of team ceremonies. Teams decide how often they will have demo days. The purpose is to share recent work of that group, with other members of the org. Demo days are a critical part to establishing accountability, feedback loops, and proper iteration. These events often occur every 2 - 4 weeks depending on agile structure and team preference. At the present moment, we have zero team working agreements, zero team retrospectives, and zero demo days. These are more important to OMI's development and growth than most other activities we do. Teams are self-organizing, therefore one master working agreement, or one master demo day for OMI is out of scope and way to broad for its purposes. Imagine if we standardized demo days to just one event and all of our working groups have to squeeze in their work into that one slot. It's not sustainable. 

Lightning talks are planned public events, with guest speakers, that can have sponsors and even a theme. Lightning talks happen monthly, quarterly, or even just yearly. These are bigger events, and can take many forms. 

I think we have conflated Lightning talks with demo days. Jin and M3 offered to collaborate with OMI for our Lightning talks, our demo days are team-based and do not require any outside co-sponsors. They are a tactical piece, not a grand event.
